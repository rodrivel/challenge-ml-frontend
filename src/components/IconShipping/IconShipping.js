import React from 'react';
import iconFreeShipping from '../../assets/ic_shipping.png';
import iconFreeShipping2x from '../../assets/ic_shipping@2x.png.png';

const IconShipping = () => <img width={20} srcSet={`${iconFreeShipping} 18w, ${iconFreeShipping2x} 36w`} src={iconFreeShipping2x} alt="Ícono envío" />;

export default IconShipping;
