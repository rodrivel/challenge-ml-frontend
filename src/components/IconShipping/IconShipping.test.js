import React from 'react';
import { shallow } from 'enzyme';
import IconShipping from './IconShipping';

it('expect to render IconShipping component', () => {
  expect(shallow(<IconShipping/>)).toMatchSnapshot();
});