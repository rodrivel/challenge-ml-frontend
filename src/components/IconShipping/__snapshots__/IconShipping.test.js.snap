// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render IconShipping component 1`] = `ShallowWrapper {}`;
